db/migrate/20170323225139_add_documents.rb

```diff
@@ -7,7 +7,10 @@ def up
       t.timestamps
     end
 
-    add_attachment :documents, :file
+    add_column :documents, :documents_file_name, :string
+    add_column :documents, :documents_content_type, :string
+    add_column :documents, :documents_file_size, :integer
+    add_column :documents, :documents_updated_at, :datetime
```

Comment:
I think you'll have to add the `documents` table again if we're going to break this out from the []String column in `snap_applications`. This is an old migration that ends up getting dropped in `db/migrate/20170728213954_drop_unused_database_tables.rb`

I'd recommend not doing *this* part until that can all be whipped into shape in one atomic commit.